pkg/reconciliation/handler.go

```diff
@@ -121,6 +121,8 @@ func (rc *ReconciliationContext) IsValid(dc *api.CassandraDatacenter) error {
 		return errs[0]
 	}
 
+	// TODO Verify if we can expand the PVC or should we reject changes
```

Comment:
**question:** we're not doing it in this PR? Any potential problem if we don't do the checks like the cluster breaking?

Reply:
We do check, this is just another place where I was planning to do the same check previously. I'll remove this comment.